viz.Rmd

```diff
@@ -176,47 +275,123 @@ To add axis labels, we use the `xlab` and `ylab` functions. To change the font s
 \index{ggplot!xlab,ylab}
 \index{ggplot!theme}
 
-```{r 03-data-co2-line-2, warning=FALSE, message=FALSE, fig.cap = "Line plot of atmospheric concentration of CO2 over time with clearer axes and labels"}
-co2_line <- ggplot(co2_df, aes(x = date_decimal, y = ppm)) +
+```{r 03-data-co2-line-2, warning=FALSE, message=FALSE, fig.cap = "Line plot of atmospheric concentration of CO$_{2}$ over time with clearer axes and labels"}
+co2_line <- ggplot(co2_df, aes(x = date_measured, y = ppm)) +
   geom_line() +
   xlab("Year") +
   ylab("Atmospheric CO2 (ppm)") +
-  theme(text = element_text(size = 18))
+  theme(text = element_text(size = 16))
 co2_line
 ```
 
+> **Note:** 
```

Comment:
In the pdf this note is misplaced, it appears under the figure so its a little confusing -- this is probably something we need to do in a larger formatting pass (placement of figures and text), but noting it here just in case
<img width="516" alt="Screen Shot 2021-09-30 at 9 30 03 PM" src="https://user-images.githubusercontent.com/3838091/135565472-ef868173-78a2-4a6b-a8e1-bcf0d6b2663a.png">